probes/toolDependabotInstalled/impl.go

```diff
@@ -0,0 +1,41 @@
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package toolDependabotInstalled
+
+import (
+	"embed"
+
+	"github.com/ossf/scorecard/v4/checker"
+	"github.com/ossf/scorecard/v4/finding"
+	"github.com/ossf/scorecard/v4/probes/utils"
+)
+
+//go:embed *.yml
+var fs embed.FS
+
+var probe = "toolDependabotInstalled"
+
+type dependabot struct {}
+
+func (t dependabot) Name() string{
+	return "Dependabot"
+}
+func (t dependabot) Matches(tool checker.Tool) bool{
+	return t.Name() == tool.Name
+}
+
+func Run(raw *checker.RawResults) ([]finding.Finding, string, error) {
+	tools := raw.DependencyUpdateToolResults.Tools
+	var matcher dependabot
+	return utils.ToolsRun(tools, fs, probe,
+		finding.OutcomePositive, finding.OutcomeNegative, matcher)
```

Comment:
It's difficult to know what this call is doing at a glance.

Reply:
I meant more from a which parameters are what perspective, not necessarily from a what the call is trying to do perspective.